packages/cli/src/commands/doctor/healthchecks/index.ts

```diff
@@ -22,33 +26,48 @@ type Options = {
   contributor: boolean | void;
 };
 
-export const getHealthchecks = ({contributor}: Options): Healthchecks => ({
-  common: {
-    label: 'Common',
-    healthchecks: [
-      nodeJS,
-      yarn,
-      npm,
-      ...(process.platform === 'darwin' ? [watchman] : []),
-      ...(process.platform === 'win32' ? [python] : []),
-    ],
-  },
-  android: {
-    label: 'Android',
-    healthchecks: [
-      jdk,
-      androidStudio,
-      androidSDK,
-      androidHomeEnvVariable,
-      ...(contributor ? [androidNDK] : []),
-    ],
-  },
-  ...(process.platform === 'darwin'
-    ? {
-        ios: {
-          label: 'iOS',
-          healthchecks: [xcode, cocoaPods, iosDeploy],
-        },
-      }
-    : {}),
-});
+export const getHealthchecks = ({contributor}: Options): Healthchecks => {
+  let additionalChecks: HealthCheckCategory[] = [];
+
+  // Todo: maybe detached commands should take the config as an arg, thats null if running in detacted.
```

Comment:
Can you clarify what detached means in this context?

Reply:
Detached is basically when you run doctor without a local RN package.  You can run `npx react-native doctor` before you init a project.

Reply:
Basically the config is project specific.  And when running in detached mode there is no app package, and so there is no config.  The CLI already runs loadConfig to try to get the local config and pass it to the commands.  But in the case of doctor, since it can run without an app package, it doesn't have the config passed to it currently.  The comment is saying that that means we end up loading the config twice in the non-detached case.  Once by the main part of the CLI.  Then again within doctor, since the original wasn't passed though. -- I'll open an issue on this and remove the todo.